### PR TITLE
Ask for sudo password if needed

### PR DESCRIPTION
**Pull request by JBlond**

### CODE REVIEW

**Comment by JBlond** [comment]
@iridakos Any feedback?

File: install [hunk 1]
```diff
@@ -33,9 +33,8 @@ if [[ "$OSTYPE" == "msys" ]]; then
 fi
 
 # sudo-check
-if [ $EUID -ne 0 ] && [ "$ON_WINDOWS" == false ]; then
-	_goto_install_error 'Please run this script in sudo mode or as root user'
-	exit
+iif [ $EUID -ne 0 ] && [ "$ON_WINDOWS" == false ]; then
```
Comment (iridakos) [inline install]: Can you please check if your change is working? Is this a typo?

Comment (JBlond) [reply install]: Yes it is a typo. Whoops

Comment (JBlond) [reply install]: You can change that to **if**

Comment (JBlond) [reply install]: Feel free to change it

**Comment by JBlond** [comment]
see now #50

**Pull request closed without merging**